src/websocket/index.ts

```diff
@@ -191,21 +209,28 @@ export const onConnection = async (socket: WebSocket.WebSocket): Promise<void> =
 
   socket.on('close', (code, reason) => {
     // Clear timeout on close
-    clearTimeout(timeoutId)
+    clearTimeout(timeoutId);
 
     // Decrement connection counter
-    activeConnections--
+    activeConnections--;
```

Comment:
I dont think the right formatting tool is being used. Semi colons are not part of our standard practices

Reply:
changed